feat(rpc-agent): allow rpc spaghetti (#126)
ForestAdmin/forestadmin-experimental: packages/datasource-rpc/README.md

# Installation

- install the package `@forestadmin-experimental/datasource-rpc`.

> - give the URI, an authsecret used to auth between RPC agent and the gateway

Reviewer: ```suggestion
- Provide the URI and the auth secret used for authentication between the RPC agent and the gateway.
```
(And I maj on the previous line)

const agent = createAgent(options).addDataSource(
  createRpcDataSource({
    uri: 'http://localhost:3352',
    authSecret: process.env.AUTH_SECRET,
  }),
);
```

# Deal with RPC relationship

> We preconize creating relationship between RPC agents on your gateway.
> But if you want to specify relationship in another way, you should use the given plugin on the gateway.

Reviewer: ```suggestion
We recommend creating a relationship between RPC agents on your gateway. However, if you want to specify the relationship in another way, you should use the provided plugin on the gateway.
```
